pacrunner.go

```diff
@@ -202,17 +200,52 @@ func myIpAddress(call otto.FunctionCall) otto.Value {
 	return toValue("127.0.0.1")
 }
 
+func myIpAddressEx(_ otto.FunctionCall) otto.Value {
+	// https://chromium.googlesource.com/chromium/src/+/ee43fa5328856129f46566b2ea1be5811739681c/net/docs/proxy.md#resolving-client_s-ip-address-within-a-pac-script-using-myipaddressex
+	public := []string{"8.8.8.8", "2001:4860:4860::8888"}
+	if ips := probeRoutes(public); ips != "" {
+		return toValue(ips)
+	}
+	if ips := resolveHostname(true); len(ips) > 0 {
+		var b strings.Builder
+		for _, ip := range ips {
+			b.WriteRune(';')
```

Comment:
```
for i, ip := range ips {
    if i > 0 {
        b.WriteRune(';')
    }
```

It's pretty simple to not put the leading `;` on in the first place, removing the need to strip it off again.

Reply:
Ok, my original thinking was that trimming the leading `;` would save having to do the check inside the loop. But I'll do it this way so that people reading it don't think it's a bug, at least initially.

Reply:
I used strings.TrimPrefix() because it handles the `len(ips) > 0` case, but as you've pointed out that's never going to happen :facepalm: